mmv1/products/dlp/InspectTemplate.yaml

```diff
@@ -71,6 +76,16 @@ properties:
     name: 'displayName'
     description: |
       User set display name of the inspect template.
+  - !ruby/object:Api::Type::String
```

Comment:
We usually omit these fields from resources. Is there any reason they need to be included in this one?

Reply:
There is no specific reason to add these fields. I've added these fields so that they could be useful, if at all. If there is no relevance, then I can go ahead and exclude these fields.

Reply:
I've excluded `createTime` and `updateTime` output-only fields from the resource.